README.md

```diff
@@ -82,17 +82,53 @@ node index config=collector.config.json pairs="COINBASE:BTC-USD,BITSTAMP:btcusdt
 
 ## How to install: Docker
 
+### Setup
+Here are the preliminary steps to follow before building and starting your containers.
+
+Create a config file based on the existing template living in the project's root folder:
+`cp config.json.example config.json`
+
+Make sure to modify the `Dockerfile` accordingly, by replacing:
+`COPY  config.json.example ${WORKDIR}`
+by 
+`COPY  config.json ${WORKDIR}`
+
+## Running Docker
 ```
 ➜ docker-compose build
 ➜ docker-compose up -d
 ```
-This will give you a running server on <http://127.0.0.1:3000> with mounted `./data` volume.
+This will give you a running server on <http://127.0.0.1:3000> with mounted `./data` volume for persistence.
 
 See `./env` file for some basic configuration.
 
-Watch logs using `docker logs -f st-server`.
+## InfluxDB
+
+To start using InfluxDB, set `storage=[ influx ]` in `config.json`.
+
+Please note: InfluxDB auth is disabled on Docker. User is root and there is no password.
+
+### Running server locally and InfluxDB on Docker
 
-Uncomment `influx` part in `docker-compose.yml` and set `STORAGE=influx` in `.env` to start using influxdb as a storage.
+In `config.json`:
+- Set `InfluxHost` to 'localhost'
+  
+In `.env` file:
+- Set `INFLUX_PORT` to config.json `influxPORT` value
```

Comment:
in .env file it says we have to use `aggr-influx` in order to use influx, but here you say I need to set to "localhost" 🤔

Reply:
change
- Set `INFLUX_PORT` to config.json `influxPORT` value

to 
- Set `INFLUX_PORT` to config.json `influxPort` value

Btw now that I test it, I have  "localhost" as influxPort variable (in the config.json) and "aggr-influx" for the INFLUX_PORT variable (in the .env)
Looks like it's working that way so not sure what you meant by that in the readme ?